tasks/task134_winowhy_reason_generation.json

```diff
@@ -0,0 +1,316 @@
+
+{
+    "Contributors": [
+        "Xinran Zhao",
+        "Hongming Zhang",
+        "Yangqiu Song"
+    ],
+    "Source": [
+        "winowhy"
+    ],
+    "Categories": [
+        "Answer Generation"
+    ],
+    "Definition": "In this task you need to give reasons to justify the pronoun coreference relations. The given sentences are provided by a sentence with a target pronoun and a sentence that justifies which noun phrase the pronoun refers to. Good practices involve the discussion about how the descriptions attached to the targeted pronoun relate to the noun phrase candidate in the question. The reasoning could come from one or multiple following knowledge types about commonsense: First: 'Property', the knowledge about property of objects (e.g., ice is cold). Second: 'Object', the knowledge about objects (e.g., cats have ears). Third: 'Eventuality', the knowledge about eventuality (e.g., 'wake up' happens before 'open eyes'). Forth: 'Spatial', the knowledge about spatial position (e.g., object at the back can be blocked). Fifth: 'Quantity', the knowledge about numbers (e.g., 2 is smaller than 10). Sixth: all other knowledge if above ones are not suitable. Write the sentence in natural language.",
```

Comment:
- In this task, there is no sentence that "a sentence that justifies which noun ...". 
 -  Also, "The given sentences" should be "The provided inputs" or something along that line, which ought to be corrected for all the tasks.

Reply:
Thank you very much for the suggestions. I will correct the wording for all the tasks very soon.

Reply:
> * In this task, there is no sentence that "a sentence that justifies which noun ...".
> * Also, "The given sentences" should be "The provided inputs" or something along that line, which ought to be corrected for all the tasks.

Hi @danyaljj , I have updated the definition again. I addressed the issue of the "the given sentences" and possible wrong explanation of the input sentence (i.e., a sentence that justifies...). I do think a completely wrong sentence could be a valid wrong "reason" that does not justify anything. It would be interesting if the LMs could generate completely unrelated sentences following the provided content. Human beings could definitely provide unrelated things if wrong reasons are needed. Yet I still remove it and write

Reply:
>  I do think a completely wrong sentence could be a valid wrong "reason" that does not justify anything. 

Ya totally. But the issue is: does this definition align with the current output? (are there a reasonable number of instances with "completely unrelated sentence" as their gold response?)  If not, a model that produces "a completely unrelated sentence" would be penalized.

Reply:
> > I do think a completely wrong sentence could be a valid wrong "reason" that does not justify anything.
> 
> Ya totally. But the issue is: does this definition align with the current output? (are there a reasonable number of instances with "completely unrelated sentence" as their gold response?) If not, a model that produces "a completely unrelated sentence" would be penalized.

yes, that is the issue that originated from the task construction. Since the annotations are from turkers, it is not likely many unrelated sentences would appear. I agree with you that we should remove "completely unrelated sentence" (and have done it since the last commit :>).